driver/pelican/discovery/main.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/SoftwareDefinedBuildings/bw2-contrib/driver/pelican/storage"
+	"github.com/SoftwareDefinedBuildings/bw2-contrib/driver/pelican/types"
+	"github.com/immesys/spawnpoint/spawnable"
+
+	_ "github.com/lib/pq"
+)
+
+func main() {
+	params := spawnable.GetParamsOrExit()
```

Comment:
I just used the same Spawnpoint parameter mechanism here, but let me know if we want to change this to use command line arguments instead.

Reply:
I think this will be fine